src/main/resources/application.yaml

```diff
@@ -91,7 +91,7 @@ idam:
     microservice: rd_caseworker_ref_api
     url: ${S2S_URL:http://rpe-service-auth-provider-aat.service.core-compute-aat.internal}
   s2s-authorised:
-    services: ${CRD_S2S_AUTHORISED_SERVICES:rd_caseworker_ref_api,rd_professional_api,am_org_role_mapping_service}
+    services: ${CRD_S2S_AUTHORISED_SERVICES:rd_caseworker_ref_api,rd_professional_api,am_org_role_mapping_service,iac}
```

Comment:
added iac in flux ?

Reply:
@shreedhar-hmcts I couldn't see flux for crd. Can you please confirm if it exists.
I had the similar doubt as well.

Reply:
Added Flux config in PR https://github.com/hmcts/cnp-flux-config/pull/7790

Reply:
Good